### PR TITLE
Adding kubeconfig to packages cli command

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/framework/cluster.go [hunk 1]
```diff
@@ -921,11 +921,13 @@ func (e *ClusterE2ETest) SetPackageBundleActive() {
 	if err != nil {
 		e.T.Fatalf("Error getting PackageBundle: %v", err)
 	}
+	os.Setenv("KUBECONFIG", kubeconfig)
 	if pbc.Spec.ActiveBundle != pb[0].ObjectMeta.Name {
 		e.RunEKSA([]string{
 			"upgrade", "packages",
 			"--bundle-version", pb[0].ObjectMeta.Name, "-v=9",
 			"--cluster=" + e.ClusterName,
+			"--kubeconfig" + e.kubeconfigFilePath(),
```
Comment (TerryHowe) [inline test/framework/cluster.go:930]: These values are the same. Do we need to set the env and pass in the variable?

Comment (jonahjon) [reply test/framework/cluster.go:930]: probably not, but I'm not sure which one should take precedent here. Maybe I'll just remove the flag, and keep the ENV var.

Comment (jonahjon) [reply test/framework/cluster.go]: Fixed in [a9154b7](https://github.com/aws/eks-anywhere/pull/4254/commits/a9154b79d80196a52f479fa590680420903ea3e9)

**Review by TerryHowe — APPROVED** [review]
/lgtm

We still want this?

**Review by TerryHowe — COMMENTED** [review]
/approve

2 way door, I can't see harm in this.

**Pull request merged**